test/features/offline_mode/woker_queue_test.dart

```diff
@@ -0,0 +1,73 @@
+@TestOn('vm')
```

Comment:
Can you add more concurrent actions test cases?
- Add and Delete at the same time
- Edit and Get the same item at the same time
- ....